assets/js/bsf-custom-fonts.js

```diff
@@ -64,11 +64,20 @@
 		},
 	}
 
-	var addButton = $('#repeater').find('.repeater-add-btn');
+	function remove_button_disble(){
```

Comment:
@shubhw12 Please update the string to `remove_button_disable`

Reply:
@premanshup I have resolved this change